pyphare/pyphare/pharesee/run/run.py

```
@@ -65,30 +67,28 @@ def _get_hier(h):
+        interpolator is a callable scipy interpolator and finest_coords is a
+        tuple of 1d coordinate arrays at the finest grid resolution.
         """
         if merged:
             domain = self.GetDomainSize()
             dl = self.GetDl(time=time)
-
-            # assumes all qties in the hierarchy have the same ghost width
-            # so take the first patch data of the first patch of the first level....
             nbrGhosts = list(hierarchy.level(0).patches[0].patch_datas.values())[
                 0
             ].ghosts_nbr
```

Ruff (0.15.20)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)